test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeUnsafeTests.java

```diff
@@ -88,4 +92,50 @@ static public void testPassingNullToIsFlattenedThrowsException() throws Throwabl
 			myUnsafe.isFlattened(null);
 		});
 	}
+
+	@Test
+	static public void testValueHeaderSizeOfValueType() throws Throwable {
+		List<String> arguments = ManagementFactory.getRuntimeMXBean().getInputArguments();
+		if (arguments.contains("-Xnocompressedrefs")) {
```

Comment:
You can store the reference mode into a static boolean field in this class, which could be used for tests added in the future if needed.

Reply:
addressed in 780435c